posthog/models/person/__init__.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any, List, Optional
```

Comment:
This seems like a pretty not okay pattern to me. There's a few reasons:

1. `__init__.py` is reserved for code running package initialisation (and is thus usually empty) https://docs.python.org/3/tutorial/modules.html#packages
2. All code in __init__ runs on first initialisation of module, which is usually first place where its called from. This is usually fine, except a few gnarly cases where you have no recourse left for import errors.

3. But most importantly: When I now want to search for a model while writing code, I need to remember to go into the __init__.py file, instead of person.py file, which can be a bit annoying


If the issue was with having imports change names everywhere, we can simply change this file to:

`from person import PersonManager, Person`

Thoughts?

Reply:
do you mean keeping something like 

/person
     __init__.py 
     person.py
     util.py

Reply:
yes exactly

Reply:
.. or getting rid of util entirely, and keeping these functions in the model file as well?

Reply:
lol we _could_ do that but that's a lot of functionality in one file

Reply:
hmmm, yeah, agreed, lots of things coming from the ee folders. Ok, just the above then!